save-orchestrator/src/main/kotlin/com/saveourtool/save/orchestrator/controller/AgentsController.kt

```diff
@@ -56,7 +56,14 @@ class AgentsController(
     private val configProperties: ConfigProperties,
     @Qualifier("webClientBackend") private val webClientBackend: WebClient,
 ) {
-    private val tmpDir = Paths.get(configProperties.testResources.tmpPath).createDirectories()
+    // Somehow simple path.createDirectories() doesn't work on macOS, probably due to Apple File System features
```

Comment:
WDYM by `doesn't work`? Does it throw an exception?

Reply:
Yes, especially for `Path("/tmp").createDirectories()` I got `FileAlreadyExistsException`

Reply:
According to the docs, `Files.createDirectories`, which is  actually used by the Kotlin extension, throws `FileAlreadyExistsException – if dir exists but is not a directory (optional specific exception)`. Maybe, tmpfs is implemented differently on Mac?

Reply:
Probably it is so